### PR TITLE
Change ActiveRecordEncryption variable to be more explicit

### PR DESCRIPTION
**Pull request by ClearlyClaire**
- change the error to mention Mastodon rather than ActiveRecord encryption, making it explicit that it is a Mastodon requirement and not something random in the stack
- `abort` instead of `raise`ing, to avoid printing a confusing stack trace
- override the `db:encryption:init` task to output something that can be copied under the right format

### CODE REVIEW

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
+# We are providing our own task with our own format
+Rake::Task['db:encryption:init'].clear
```
Comment (mjankowski) [inline lib/tasks/db.rake:4]: It feels sort of odd to replace the framework task instead of adding a new task?

File: lib/tasks/db.rake [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
 namespace :db do
+  namespace :encryption do
+    desc 'Generate a set of keys for configuring Active Record encryption in a given environment'
+    task init: :environment do
+      puts <<~MSG
+        Add these environment variables to your Mastodon environment:#{' '}
+
+        ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY=#{SecureRandom.alphanumeric(32)}
+        ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT=#{SecureRandom.alphanumeric(32)}
+        ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY=#{SecureRandom.alphanumeric(32)}
```
Comment (ThisIsMissEm) [inline lib/tasks/db.rake:15]: Should these values be quoted?

**Review by mjankowski — APPROVED** [review]
Left a minor naming comment about what to call the task, but overall the changes are a good improvement for communicating how to add this.

File: lib/tasks/db.rake (hunk 2 above)
Comment (ClearlyClaire) [reply lib/tasks/db.rake:15]: I don't think so, `SecureRandom.alphanumeric(32)` values will always be safe for shell, dotenv and Docker env

File: lib/tasks/db.rake (hunk 1 above)
Comment (ClearlyClaire) [reply lib/tasks/db.rake:4]: I guess we could add another task with another name, but this one serves the same purpose. I'm open to suggestions though.

**Review by renchap — APPROVED** [review]

**Pull request merged**